app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HomepageController < ApplicationController
+
+  def index
+    @netflixes = Netflix.all.sort_by {|n| n.ranking}.reverse!
+    @capstones = Capstone.all.sort_by {|c| c.ranking}.reverse!
+    @books = Book.all.sort_by {|b| b.ranking}.reverse!
```

Comment:
For these lines (and similar ones in the other controllers) we should use the ActiveRecord form of this code, which allows the database to do the sorting work for us:
```ruby
@books = Book.all.order(ranking: :desc)
```
This tells the database to return all of the `Book` models, sorted by the `ranking` attribute in _descending_ order.